src/overlays/actors/ovl_En_Tab/z_en_tab.c

```diff
@@ -67,15 +67,15 @@ s32 D_80BE1A0C[] = {
 };
 
 ActorInit En_Tab_InitVars = {
-    /**/ ACTOR_EN_TAB,
-    /**/ ACTORCAT_NPC,
-    /**/ FLAGS,
-    /**/ OBJECT_TAB,
-    /**/ sizeof(EnTab),
-    /**/ EnTab_Init,
-    /**/ EnTab_Destroy,
-    /**/ EnTab_Update,
-    /**/ EnTab_Draw,
+    ACTOR_EN_TAB,
+    ACTORCAT_NPC,
+    FLAGS,
+    OBJECT_TAB,
+    sizeof(EnTab),
+    EnTab_Init,
+    EnTab_Destroy,
+    EnTab_Update,
+    EnTab_Draw,
```

Comment:
This is still showing a diff, could you bring back the `/**/` comments? It's there to help with formatting